src/operator/nn/batch_norm.cc

```diff
@@ -567,10 +567,9 @@ axis to be the last item in the input shape.
 Both ``gamma`` and ``beta`` are learnable parameters. But if ``fix_gamma`` is true,
 then set ``gamma`` to 1 and its gradient to 0.
 
-Note::
-
-When fix_gamma is set to True, no sparse support is provided. If fix_gamma is set to False,
-the sparse tensors will fallback.
+.. Note::
+  When fix_gamma is set to True, no sparse support is provided. If fix_gamma is set to False,
```

Comment:
I think consistent use of the double ticks would be good here.
```suggestion
  When ``fix_gamma`` is set to True, no sparse support is provided. If ``fix_gamma is`` set to False,
```